lib/ui/window/platform_configuration.cc

```diff
@@ -424,6 +442,20 @@ void PlatformConfiguration::CompletePlatformMessageResponse(
   response->Complete(std::make_unique<fml::DataMapping>(std::move(data)));
 }
 
+void PlatformConfiguration::CompleteKeyDataResponse(uint64_t response_id, bool handled) {
+  if (!response_id) {
```

Comment:
Nit: If you want to compare an int to zero, use `== 0` rather than `!` so that it doesn't look like a bool or pointer.